node/src/auth/Auth.js

```diff
@@ -1,5 +1,8 @@
 const AUTH = WeDeploy.auth('auth-taste.wedeploy.io');
 
+// Data of the user signed in.
+export const currentUser = AUTH.currentUser;
```

Comment:
I'm doing this since if `Auth.js` is imported elsewhere I'll have to do `Auth.AUTH.currentUser`. Unless it's better to configure the import statement to clean this up instead.